### PR TITLE
Feature/collision avoidance

### PR DESCRIPTION
**Pull request by alecive**
I am merging now to fix travis builds. @brahmgardner please update all branches on the Baxter robot!

### CODE REVIEW

**Comment by alecive** [comment]
@brahmgardner  after the merge, you can keep working on this branch without issues.

File: lib/include/react_controller/baxterChain.h [hunk 1]
```diff
@@ -54,7 +54,7 @@ class BaxterChain : public KDL::Chain
      *
      * return:     true if successful, false if error
      */
-    bool JntToCart(const KDL::JntArray& _q_in, KDL::Frame& _p_out, int _segNr=-1);
+    bool JntToCart(const KDL::JntArray& _q_in, KDL::Frame& _p_out, int seg_nr=-1);
```
Comment (brahmgardner) [inline lib/include/react_controller/baxterChain.h]: we would want _seg_nr right?

Comment (alecive) [inline lib/include/react_controller/baxterChain.h]: Yes I believe so. What do you mean? I just changed its name to have similar naming to `JntToJac`

Comment (brahmgardner) [inline lib/include/react_controller/baxterChain.h]: i was just talking about the underscore at the beginning. sorry! :)

**Pull request merged**